1.1/geo.ttl

```diff
@@ -231,26 +232,101 @@ xsd:date a rdfs:Datatype .
 	rdfs:domain :Feature ;
 	rdfs:range :Geometry ;
 	dc:contributor "Matthew Perry" ;
-	dc:creator "OGC GeoSPARQL 1.0 Standard Working Group" ;
+	dc:creator "OGC GeoSPARQL Standard Working Group" ;
 	dc:date "2011-06-16"^^xsd:date ;
 	dc:description """A spatial representation for a given feature."""@en ;
 	rdfs:comment """A spatial representation for a given feature. """@en ;
-	rdfs:isDefinedBy <http://www.opengis.net/ont/geosparql> , <http://www.opengis.net/spec/geosparql/1.0> ;
+	rdfs:isDefinedBy <http://www.opengis.net/ont/geosparql> , <http://www.opengis.net/def/geosparql/1.1> ;
 	rdfs:label "hasGeometry"@en ;
 	skos:definition """A spatial representation for a given feature."""@en ;
 	skos:prefLabel "hasGeometry"@en .
 # 
+# http://www.opengis.net/ont/geosparql#hasArea
+
+:hasArea a owl:ObjectProperty ;
+	rdfs:domain :Feature ;
+	rdfs:range :SpatialMeasure ;
+	dc:contributor "Simon J.D. Cox / Nicholas J. Car" ;
+	dc:creator "OGC GeoSPARQL Standard Working Group" ;
+	dc:date "2020-01-04"^^xsd:date ;
+	dc:description """The area of a two-dimensional Feature, expressed as a Spatial Measure."""@en ;
+	rdfs:comment """The area of a two-dimensional Feature, expressed as a Spatial Measure."""@en ;
+	rdfs:isDefinedBy <http://www.opengis.net/ont/geosparql> , <http://www.opengis.net/def/geosparql/1.1> ;
+	rdfs:label "hasGeometry"@en ;
+	skos:definition """The area of a Feature, expressed as a Spatial Measure."""@en ;
+	skos:prefLabel "has area"@en .
+# 
+# http://www.opengis.net/ont/geosparql#hasLength
+
+:hasLength a owl:ObjectProperty ;
+	rdfs:domain :Feature ;
+	rdfs:range :SpatialMeasure ;
+	dc:contributor "Simon J.D. Cox / Nicholas J. Car" ;
+	dc:creator "OGC GeoSPARQL Standard Working Group" ;
+	dc:date "2020-01-04"^^xsd:date ;
+	dc:description """The length of a one-dimensional Feature, expressed as a Spatial Measure."""@en ;
```

Comment:
Does length only apply to 1-D features?

Reply:
I did think so but perhaps you are thinking of perimeter of a 2D think length?

Applied commit https://github.com/opengeospatial/ogc-geosparql/pull/72/commits/13ce7a17081b325872b4b598c69e5f44f88cd7b0, does that cover it?

Reply:
Length is commonly used to denote the extent of the longest side or dimension. Length of a swimming pool, or of a football field, lake, peninsula, etc. Perhaps such common semantics is unhelpful here, but I wanted to ask the question at least.

Reply:
I'm ok with a multi-dimentional possible interpretation of length (or any other property) but I think that "Length ... used to denote the extent of the longest side or dimension" is a special (Quantity Kind?) length and is really just length but pertaining to something in particular - still length. Sounds like a property with qualifiers, roles, etc., or even a different target Feature.